packages/config/package.json

```
@@ -0,0 +1,48 @@
+{
+  "name": "@redwoodjs/config",
```

Just spoke to @thedavidprice; we think config is too broad, and there's other config in the internal package (like build) that this package doesn't include. We want to narrow it down a bit:

```suggestion
  "name": "@redwoodjs/project-config",
```

---

(I'd say don't make this change yet, just taking notes from discussion)